lib/sidekiq/web/helpers.rb

```diff
@@ -148,6 +148,29 @@ def processes
       @processes ||= Sidekiq::ProcessSet.new
     end
 
+    # Sorts processes by hostname following the natural sort order so that
+    # 'worker.1' < 'worker.2' < 'worker.10' < 'worker.20'
+    # '2.1.1.1' < '192.168.0.2' < '192.168.0.10'
+    def sorted_processes
+      @sorted_processes ||= begin
+        return processes unless processes.all? { |p| p['hostname'].present? }
```

Comment:
Is `present?` an ActiveSupport thing? Can you verify your method runs in a bare IRB session?